Reverse_in_groups_LL.py

```diff
@@ -0,0 +1,98 @@
+'''
+Given a linked list of size N. 
+Reverse every k nodes (where k is an input to the function) in the linked list.
+
+Example:
+
+Input:
```

Comment:
Sample I/O would look better at EOF
And you also need to mention time complexity (above sample IO)

Reply:
> Sample I/O would look better at EOF
> And you also need to mention time complexity (above sample IO)

Made the changes.. please have a look